src/jsonl/index.js

```diff
@@ -0,0 +1,117 @@
+'use strict';
+
+const fsBase = require('fs');
+const fs = fsBase.promises;
+const readline = require('readline');
+const { dirname, basename, join } = require('path');
+
+const delimeter = '\n';
+
+const serialize = async (obj) => JSON.stringify(obj) + delimeter;
+const deserialize = async (str) => JSON.parse(str);
+
+const append = async (path, ...objects) => {
+    const fileHandle = await fs.open(path, 'a');
+    if (fileHandle === undefined) throw new Error('Could not open the file!');
+    try {
+        for (const obj of objects) {
+            const string = await serialize(obj);
+            await fileHandle.write(string);
+        }
+    } finally {
+        await fileHandle.close();
```

Comment:
await inside finally is not a good idea

Reply:
Can you please explain why is that a bad idea or refer to the place where I can learn that? Not arguing in any way, just curious because I've seen this approach in multiple places on the internet (very well could be bad examples).

Reply:
1. Here `close()` may return rejected promise in it will lead to `unhandledRejection`
2. Method `close()` may throw and it will also lead to `unhandledRejection`
3. Event `unhandledRejection` will appear after async function return, so sync code after call will be executed first (fefore) finally section and it is not obvious behavior.